### ISSUE TITLE
Issues involving loading configuration via Json

### ISSUE THREAD

**Issue by ExcaliburVT** [mmajcica/DeploySsrs#25]
[Folder]:new() doesn't have a 3 parameter overload, assume it was refactored and not tested with the JSON Version

Two calls to the += operator were not valid and did not appear to be needed.

Converted the true/false to boolean just like the GetXmlFolderItems method.

Added a Json version of the example ReportConfiguration file

**Comment by mmajcica** [comment]
First of all, thanks.
You are right assuming that the JSON config was not much tested. Initially, the configuration should have only be supplied via the JSON config file, however, during the development on the customers site I realized that XML is going to be the format, so I decided to implement the necessary and I tested thoroughly only XML config files.
I did a quick review of the changes you made, if you do agree with my comments, I'll merge and deploy a new version today.
